src/stache/includes/cli/lint-format.html

```diff
@@ -0,0 +1 @@
+<stache-code>--format [string]</stache-code> — Sets the tslint format option. This flag is passed directly <stache-code>tslint</stache-code> library.  See <a href="https://palantir.github.io/tslint/formatters/">their documentation</a> for a list of formatters.
```

Comment:
Should tslint format be `tslint format`? Also, the second sentence is missing "to the" as in "This flag is passed directly to the `tslint` library."

Reply:
Addressed in separate PR